src/userDashboard/dataFormat/BarGraphDataFormat.js

```diff
@@ -0,0 +1,221 @@
+import { eachMonthOfInterval, subDays, format, getYear, getMonth, eachDayOfInterval, subYears } from 'date-fns';

```

Comment:
a comment at the beginning of every file that explains what the file does would be good.
i can't figure out the point of some of the files otherwise (e.g. this one, but also the commonDataFormat);

Reply:
added extra documentation/comments for relevant functions.